taworks/taform/views.py

```diff
@@ -689,6 +688,11 @@ def instructor_ranking(request, hash):
     num_apps = apps.count()
     num_students = students.count()
 
+    if (apps.count() == 0):
```

Comment:
noApps = True if num_apps == 0 else False

^ ternary, this line can replace 691-694. you don't have to change it just wanted to teach you another tool :)